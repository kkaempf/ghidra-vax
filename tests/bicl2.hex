ca 8f 80 00 00 00 9f 00 00 08 20
