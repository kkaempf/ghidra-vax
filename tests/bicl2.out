0x00000000: BICL2 I^#0x80, @#0x20080000
